### PR TITLE
Fix: use environment variables to hide secret and client id

### LINKED ISSUE #71: Rotate Google API secrets and start using secrets
https://github.com/podkolzzzin/AcademicHub/blob/main/Identity/Identity/Program.cs#L29 - you literally shared google secrets on public that everybody can use it. It is serious vulnerability: any hacker can use this secrets for some scam and AcademicHub would be abused in it.

Even if you remove them they would be still visible in history.

So you have to rotate them and start using from secrets.

### PR DESCRIPTION
**Pull request by YaroslavKSE**
@podkolzzzin I have use environment variables to hide secret and client id. However, I can't configure secrets as I haven't got permissions. You can add them by yourself or I can use your pc to add them. Closes #71

### CODE REVIEW

**Comment by podkolzzzin** [comment]
I have transferred project to the organization and granted you admin access.
Hope it works.

**Comment by YaroslavKSE** [comment]
> I have transferred project to the organization and granted you admin access. Hope it works.

Thanks, I can add secretes now

File: Identity/.idea/config/applicationhost.config [hunk 1]
```diff
@@ -0,0 +1,993 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Bardin08) [inline Identity/.idea/config/applicationhost.config:1]: Please remove this, as it's an IDE-related file. It's some Rider's cache

**Review by Bardin08 — CHANGES_REQUESTED** [review]
@YaroslavKSE PR looks good to me, but:
- Remove Rider's cache from it
- Please confirm that:
	- Google OAuth secrets rotated
	- Google OAuth secrets added to the repos SECRETS (not variables)

When done please ping me and I will approve this PR

**Comment by Bardin08** [comment]
@YaroslavKSE I closed this PR because it didn't resolve security issues, as variables are still in the codebase. Instead, please check the approach defined in the #90. There is used a well-known feature called user secrets.

### Useful links:
- https://blog.jetbrains.com/dotnet/2023/01/17/securing-sensitive-information-with-net-user-secrets/

**Pull request closed without merging**